config/initializers/voyager_helpers.rb

```diff
@@ -1,5 +1,4 @@
 require 'yaml'
-require_relative '../../voyager_helpers/lib/voyager_helpers'
 include VoyagerHelpers::OracleConnection
 
 vh_config = YAML.load_file("#{File.dirname(__FILE__)}/../voyager_helpers.yml")
```

Comment:
There are instructions with the new gem. Maybe just ref. those?

https://github.com/pulibrary/voyager_helpers/blob/development/README.md#configuration

-Js

Sent via mobile. Please excuse typos, brevity, etc.